models/share.php

```diff
@@ -11,6 +11,12 @@ public function add(){
 		$post = filter_input_array(INPUT_POST, FILTER_SANITIZE_STRING);
 
 		if($post['submit']){
+
+			if($post['title'] == '' || $post['body'] == ''|| $post['link'] == '' ){
```

Comment:
Always use strict type checks.
```suggestion
			if($post['title'] === '' || $post['body'] === '' || $post['link'] === '' ){
```